src/lib/constants/postCard-config.ts

```diff
@@ -0,0 +1,14 @@
+import { IPost, DirectionEnum, PostStatus, PostTypeEnum } from '../types';
+
+export const mockData: IPost = {
```

Comment:
for such a constants pls use `MOCK_DATA` case

Reply:
fixed